x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -96,6 +108,22 @@ async function createSecurityProject(
     product_types: productTypes,
   };
 
+  log.info(`Kibana override flag equals to ${process.env.KIBANA_MKI_USE_LATEST_COMMIT}!`);
+  if (
+    process.env.KIBANA_MKI_USE_LATEST_COMMIT &&
+    process.env.KIBANA_MKI_USE_LATEST_COMMIT === '1'
+  ) {
+    const kibanaOverrideImage = `${process.env.BUILDKITE_COMMIT?.substring(0, 12)}`;
+    log.info(
+      `Overriding Kibana image in the MKI with docker.elastic.co/kibana-ci/kibana-serverless:sec-sol-qg-${kibanaOverrideImage}`
+    );
+    body.overrides = {
+      kibana: {
+        docker_image: `docker.elastic.co/kibana-ci/kibana-serverless:sec-sol-qg-${kibanaOverrideImage}`,
```

Comment:
Same as: https://github.com/elastic/kibana/pull/171859/files#r1479479038

Reply:
This (docker.elastic.co/kibana-ci/kibana-serverless) is hardcoded in other places as well e.g: https://github.com/elastic/kibana/blob/24d36198c7e89bd1d6c216a29bd690160c0b1707/.buildkite/scripts/steps/artifacts/docker_image.sh#L16

I guess its not a problem to be hardcoded for us as well. If you insist though, I can move it definitely to vault.

Reply:
I guess it is not then :)